spec/sailfish_iptables_dbus_interface_description.xml

```diff
@@ -0,0 +1,464 @@
+ 					Remove removes the rule from iptables.
+ protocol	s 		tcp/udp, case insensitive (sctp not supported by sailfishos
+ 					iptables yet), usually defaults to TCP as tcp protocol is
+ 					defined first in /etc/services. Can be empty if service/
+ 					port is found from /etc/services.
+
+The methods are named with following convention:
+	Permission|Direction|Parameters
+	
+Permission:
+	Allow			Add rule to iptables with "-j ACCEPT"
+	Deny			Add rule to iptables with "-j DROP"
+	
+Direction:
+	Incoming		Add rule to chain INPUT of filter table
+	Outgoing		Add rule to chain OUTPUT of filter table
+	
+Parameters:
+	Ip			IP address as input
+	With			Combine two parameters
+	Port			Port parameter (integer q)
+	PortRange		Port or port range as parameter (string s)
+	Service			Service name as parameter
+	
+Each method sends signal RuleChanged if the rule was added to iptables.
+-->
+
+<!-- Add IP address or network to filter table INPUT chain as ACCEPT. -->
+<method name="AllowIncomingIp">
+<arg name="ip" type="s" direction="in"/>
+<arg name="operation" type="s" direction="in"/>
+<arg name="result" type="q" direction="out"/>
+<arg name="string" type="s" direction="out"/>
+</method>
+
+<!-- Add IP address or network with port to filter table INPUT chain as ACCEPT.
+Parameter protocol can be omitted if port is of some known service (/etc/services)
+-->
+<method name="AllowIncomingIpWithPort">
+<arg name="ip" type="s" direction="in"/>
+<arg name="port" type="q" direction="in"/>
+<arg name="protocol" type="s" direction="in"/>
+<arg name="operation" type="s" direction="in"/>
+<arg name="result" type="q" direction="out"/>
+<arg name="string" type="s" direction="out"/>
+</method>
+
+<!--
+Add IP address or network with port range (e.g., 2000:3000,) to filter table
+INPUT chain as ACCEPT.
+-->
+<method name="AllowIncomingIpWithPortRange">
+<arg name="ip" type="s" direction="in"/>
+<arg name="port" type="s" direction="in"/>
+<arg name="protocol" type="s" direction="in"/>
+<arg name="operation" type="s" direction="in"/>
+<arg name="result" type="q" direction="out"/>
+<arg name="string" type="s" direction="out"/>
+</method>
+
+<!--
+Add port with address any (0.0.0.0) to filter table INPUT chain as ACCEPT.
+Parameter protocol can be omitted if port is of some known service (/etc/services)
+-->
+<method name="AllowIncomingPort">
+<arg name="port" type="q" direction="in"/>
+<arg name="protocol" type="s" direction="in"/>
+<arg name="operation" type="s" direction="in"/>
+<arg name="result" type="q" direction="out"/>
+<arg name="string" type="s" direction="out"/>
+</method>
+
+<!-- Add port range (e.g., 2000:3000) with address any (0.0.0.0) to filter table
+INPUT chain as ACCEPT.
+-->
+<method name="AllowIncomingPortRange">
+<arg name="port" type="s" direction="in"/>
+<arg name="protocol" type="s" direction="in"/>
+<arg name="operation" type="s" direction="in"/>
+<arg name="result" type="q" direction="out"/>
+<arg name="string" type="s" direction="out"/>
+</method>
```

Comment:
Duplicate of `AllowIncomingIpWithPortRange`. Unnecessary.

Reply:
See prev. comment.